templates/layout.jade

```diff
@@ -6,12 +6,77 @@ html
     // Viewport mobile tag for sensible mobile support
     meta(name="viewport",content="width=device-width, initial-scale=1, maximum-scale=1")
 
+    //-
+      Stylesheets and Preprocessors
+      ==============================
+
+      You can always bring in CSS files manually with `<link>` tags, or asynchronously
+      using a solution like AMD (RequireJS).  Or, if you like, you can take advantage 
+      of Sails' conventional asset pipeline (boilerplate Gruntfile).
+
+      By default, stylesheets from your `assets/styles` folder are included
+      here automatically (between STYLES and STYLES END). Both CSS (.css) and LESS (.less)
+      are supported. In production, your styles will be minified and concatenated into
+      a single file.
+
+      To customize any part of the built-in behavior, just edit `tasks/pipeline.js`.
+      For example, here are a few things you could do:
+
+          + Change the order of your CSS files
+          + Import stylesheets from other directories
+          + Use a different or additional preprocessor, like SASS, SCSS or Stylus
+
+    // STYLES
+    // STYLES END
+
     block styles
 
   body
     block body
 
-    // The linker searches for this tag and adds js.  If there's no linker, we point to sails.io.js
+    //-
+      Client-side Templates
+      ========================
+
+      HTML templates are important prerequisites of modern, rich client applications.
+      To work their magic, frameworks like Backbone, Angular, Ember, and Knockout require
+      that you load these templates client-side.
+
+      By default, your Gruntfile is configured to automatically load and precompile
+      client-side JST templates in your `assets/templates` folder, then
+      include them here automatically (between TEMPLATES and TEMPLATES END).
+
+      To customize this behavior to fit your needs, just edit `tasks/pipeline.js`.
+      For example, here are a few things you could do:
+
+          + Import templates from other directories
+          + Use a different template engine (handlebars, jade, dust, etc.)
+          + Internationalize your client-side templates using a server-side
+            stringfile before they're served.
+
+    // TEMPLATES
+    // TEMPLATES END
```

Comment:
Hi @konstantinzolotarev, actually these three sections are from the default ejs views, and for alignment, I kept the origin information and revised them into jade syntax.

The first problem I met was that the styles cannot be injected into even pipeline.js is configured. Afterwards, I found the initial jade views are not as same as the default ejs views. Frankly, I also was wondering should this 'TEMPLATES' section be put here or not, however, after a considerable thought, I think it would be better to keep it as same as the default views (ejs) at initial views.

Please feel free to tell me your views on this matter.

Cheers.